code/alden/python/lab08.py

```diff
@@ -0,0 +1,94 @@
+import re
```

Comment:
Needs Work:
* REPL should keep looping until the user decides to end the loops.
    * User should be able to do the four CRUD without having to restart the python program.
* See me if you have any questions.